assets/src/components/vehicleRouteSummary.tsx

```diff
@@ -87,7 +87,7 @@ export const VehicleRouteSummary = ({
       className="m-vehicle-route-summary__route-variant headsign font-m-semi"
     />
 
-    {isVehicle(vehicle) && (
+    {isVehicle(vehicle) && !vehicle.isShuttle && (
```

Comment:
suggestion: Could you add a test for this shuttle-specific behavior?

Reply:
In the process 👍🏻 
Should have something in a few.